app/src/main/java/de/danoeh/antennapod/preferences/PreferenceUpgrader.java

```diff
@@ -109,5 +109,12 @@ private static void upgrade(int oldVersion, Context context) {
         if (oldVersion < 2050000) {
             prefs.edit().putBoolean(UserPreferences.PREF_PAUSE_PLAYBACK_FOR_FOCUS_LOSS, true).apply();
         }
+        if (oldVersion < 2070000) {
```

Comment:
2.7 is already in beta, so the version this will land in is 2.8 :)

Reply:
Thanks :)